### DISCUSSION TITLE
The favicon wont change despite me changing it

### DISCUSSION THREAD

**Discussion by K-arlach** [ubai-m/react-portfolio#63] [General]
![image_2024-08-26_035110577](https://github.com/user-attachments/assets/83fe836a-e809-49dc-8185-8dc221a1057f)
![image](https://github.com/user-attachments/assets/9bbc77f8-1352-4ace-bf59-4047f99ed561)
i changed the icon in the HTML file head and it stays to that yellow guy...

**Comment by ubaimutl** [comment]
Clear your browser cache or try incognito mode; it might be caching the old favicon. Also, make sure your `<link rel="icon" href="%PUBLIC_URL%/path_to_icon_in_public_folder.ico" />` is correct, and check the console for any errors related to the favicon path.

**Reply by K-arlach** [reply]
i will try thanks, can't i use a local path for it ?

**Reply by K-arlach** [reply]
Also sorry but how can i change the template inside the portfolio, i manage to resize to the way i want but i cannot seem to reduce to 1 or 2 column ?

**Reply by ubaimutl** [reply]
What do u mean? put ur icon in the /public/ folder then go to index.html and add this line

<link rel="icon" href="%PUBLIC_URL%/Logo.png" />

**Reply by ubaimutl** [reply]
> Also sorry but how can i change the template inside the portfolio, i manage to resize to the way i want but i cannot seem to reduce to 1 or 2 column ?

To change the number of columns you need to modify the index.css in /pages/portfolio:
Increase/Decrease column-width: Smaller column-width values will result in more columns, while larger values will result in fewer columns.
Adjust column-gap: Change the spacing between columns without affecting the number of columns.

**Reply by ubaimutl** [reply]
For example, if u want fewer columns with more space between them:

```css
.po_items_ho {
    column-width: 20em; 
    column-gap: 6rem; 
}

**Reply by K-arlach** [reply]
Thank you very much !

**Reply by K-arlach** [reply]
Never mind it does not fix it the icon : 
![image](https://github.com/user-attachments/assets/ee9dae71-6094-4f23-8731-29fbba8fc24f)
this is my code, the icon is in the public folder yet it doesnt display, and when i go on the about page, the yellow dude appears...

**Reply by ubaimutl** [reply]
Use this line:

```html
<link rel="icon" href="%PUBLIC_URL%/Logo.ico" />
```

If it didn’t work, copy the error from the console and paste it here.

**Reply by K-arlach** [reply]
thanks ! working !
